plugins/cilium-cni/cilium-cni.go

```diff
@@ -50,7 +50,7 @@ var (
 )
 
 func init() {
-	log.Level = logrus.DebugLevel
+	logging.SetLogLevel(logrus.DebugLevel)
```

Comment:
It's the correct 1:1 fix but do we really want to unconditionally enable debug logging for the cni plugin?

Reply:
It seems to have been there for a long time. Comparing it to the cilium-docker plugin, though, it does seem unintended. That one has a toggle https://github.com/eloycoto/cilium/blob/dde6508b118ba6c6662b94f07776cd421f9bb3a0/plugins/cilium-docker/main.go#L81-L85